src/ui/components/pages/Account/tabs/AccountK8sTab.tsx

```diff
@@ -0,0 +1,152 @@
+import { useMemo, memo } from "react";
+import { useTranslation } from "ui/i18n";
+import { AccountSectionHeader } from "../AccountSectionHeader";
+import { AccountField } from "../AccountField";
+import { useCallbackFactory } from "powerhooks/useCallbackFactory";
+import { copyToClipboard } from "ui/tools/copyToClipboard";
+import Divider from "@mui/material/Divider";
+import { makeStyles } from "ui/theme";
+import exportK8S from "js/components/mon-compte/export-credentials-k8s";
+import { assert } from "tsafe/assert";
+import { saveAs } from "file-saver";
+import { smartTrim } from "ui/tools/smartTrim";
+import { useFormattedDate } from "ui/useMoment";
+import { useSelector } from "ui/coreApi";
+import { declareComponentKeys } from "i18nifty";
+import * as jwtSimple from "jwt-simple";
+
+export type Props = {
+    className?: string;
+};
+
+export const AccountK8sTab = memo((props: Props) => {
+    const { className } = props;
+
+    const { classes } = useStyles();
+
+    const { t } = useTranslation({ AccountK8sTab });
+
+    const selectedProjectId = useSelector(
+        state => state.projectSelection.selectedProjectId,
+    );
+
+    const accessToken = useSelector(state => state.userConfigs.accessToken.value || "");
+    const availableDeploymentRegions = useSelector(
+        state => state.deploymentRegion.availableDeploymentRegions,
+    );
+    const kubernetesClusterDomain =
+        availableDeploymentRegions[0].kubernetesClusterDomain || "";
+    const kubernetesServerUrl = availableDeploymentRegions[0].kubernetes?.url || "";
+
+    const user = jwtSimple.decode(accessToken, "", true)["preferred_username"] || "";
+    const expirationTime = jwtSimple.decode(accessToken, "", true)["exp"] || "";
```

Comment:
This is not logic that should be performed in a react component but it the core.